src/app/GlobalStyles.tsx

```diff
@@ -0,0 +1,28 @@
+'use client'
```

Comment:
does `<Global />` require 'use client'?

Reply:
The server error occurs when I omit 'use client'.

```
Server Error
Error: createContext only works in Client Components. Add the "use client" directive at the top of the file to use it. Read more: https://nextjs.org/docs/messages/context-in-server-component

This error happened while generating the page. Any console logs will be displayed in the terminal window.
Call Stack
React
.yarn/__virtual__/@emotion-react-virtual-a9450e77d7/3/.yarn/berry/cache/@emotion-react-npm-11.11.3-5802f4fe7f-10c0.zip/node_modules/@emotion/react/dist/emotion-element-6bdfffb2.esm.js (14:41)
(rsc)/./.yarn/__virtual__/@emotion-react-virtual-a9450e77d7/3/.yarn/berry/cache/@emotion-react-npm-11.11.3-5802f4fe7f-10c0.zip/node_modules/@emotion/react/dist/emotion-element-6bdfffb2.esm.js
...
```

Reply:
It seems that one of the recommended approaches by the [documentation]((https://emotion.sh/docs/ssr)) is to use React's [renderToString](https://reactjs.org/docs/react-dom-server.html#rendertostring) or [renderToNodeStream](https://reactjs.org/docs/react-dom-server.html#rendertonodestream) methods 🤔